lib/commands/gesture.js

```diff
@@ -79,6 +82,13 @@ function isDoubleTap (gestures) {
   return false;
 }
 
+function isTwoFingerTap (gestures) {
+  if (gestures.length === 1 && gestures[0].action.toLowerCase() === 'twofingertap') {
```

Comment:
You're right, however given that when we _do_ migrate to the new W3C spec, and it doesn't seem to have a "double-tap" primitive, probably better to do it the way that will be more easy to map to the spec later.